bear-requirements.yaml

```diff
@@ -8,8 +8,6 @@ gem_requirements:
     version: ~>0.4.0
   fasterer:
     version: ~>0.4.1
-  haml_lint:
```

Comment:
the reason it is failing is because you are removing these dependencies from coala-bears.

Probably ... you dont have haml_lint installed, so when you run the generate script, it skips this one. (that is a bug ... we need to fix this)

The CI log shows

```console
.ci/generate_bear_requirements.py --check --update
- - gem_requirements
  - - - haml_lint
      - Missing
- - npm_requirements
  - - - write-good
      - Missing
    - - stylint
      - Missing
- - pip_requirements
  - - - git-url-parse
      - Missing
    - - cpplint
      - Missing
    - - nltk
      - Missing
    - - pylint
      - Missing
    - - memento_client
      - Missing
```

Dont bother installing all of these locally ... its not important.

You can manually edit this file, and re-add the bits which have been removed.

Update your commit, and check that your commit only adds `composer_requirements:` with the new php dependencies.